src/checkers/style.css

```diff
@@ -1,4 +1,4 @@
-body {
+/*body {
```

Comment:
Except maybe we can do it where all the html is on one page and we add like a "hidden" class to remove the things we don't want to show? Instead of the weird javascript injection that isn't very clean